index.bs

```diff
@@ -155,28 +156,38 @@ Parsing {#parsing}
 To <dfn abstract-op local-lt="parse header">obtain a response's embedder policy</dfn> given a
 [=response=] (|response|):
 
-1.  Let |policy| be "`unsafe-none`".
-
-2.  Let |header| be the result of [=header list/getting=] `Cross-Origin-Embedder-Policy` from
+1.  Let |header| be the result of [=header list/getting=] `Cross-Origin-Embedder-Policy` from
```

Comment:
You can change step 1-4 to use "[get a structured header](https://fetch.spec.whatwg.org/#concept-header-list-get-structured-header)", passing in `Cross-Origin-Embedder-Policy` and `item`.

That will give you |parsed item| directly (which, according to https://tools.ietf.org/html/draft-ietf-httpbis-header-structure-15#section-4.2.3, is a `(bare_item, parameters)` tuple.

Reply:
It seems to me we could avoid duplicated effort of figuring out how to correctly get structured headers by getting https://github.com/mikewest/corpp/pull/2 done and then doing the same thing here.

Are we ready to introduce SH parameters now? @annevk @mnot

Reply:
Parameters are part of the SH spec, which @mnot asserts elsewhere is ready for last call. I think it's reasonable to expect that they'll be part of headers going forward.

Reply:
Done.